crabs/detection_tracking/optuna.py

```diff
@@ -0,0 +1,56 @@
+from typing import Callable, Dict, Union
+
+import optuna
```

Comment:
I don't think we can call this file, optuna. As the library is called optuna. We can rename this file as `hpo.py` which I think that what people used nowadays

Reply:
I asked the Python hivemind about this, it shouldn't give an error since the namespace is different but we may want to prevent any issues by renaming the module to a private `_optuna`. 

I'll do that for now, but when we chat we can also try debugging why that is happening for you.